scripts/fastq_tool.py

```diff
@@ -0,0 +1,52 @@
+def count_gc_content(sequence: str) -> float:
+    counter = 0
+    for base in sequence:
+        if base == 'G' or base == 'C':
+            counter += 1
+    gc_content = counter / len(sequence) * 100
+    return round(gc_content, 2)
+
+
+def check_quality(seq_quality: str, threshold: float) -> bool:
+    quality_ascii = []
+    for char in seq_quality:
+        quality_ascii.append(ord(char))
+
+    quality_perc = list(map(lambda x: int(x) - 33, quality_ascii))
+    mean_quality = sum(quality_perc) / len(quality_perc)
+
+    return True if mean_quality > threshold else False
```

Comment:
Ой, не, не надо делать булеву операцию чтобы проверить какое булево значение будет в булевом результате чтобы вернуть булево значение
```suggestion
    return mean_quality > threshold 
```
:)